test/hotspot/jtreg/compiler/jvmci/TestBootstrapJVMCIOption.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2021, Red Hat, Inc. All rights reserved.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ *
+ */
+
+/*
+ * @test TestBootstrapJVMCIOption
+ * @bug 8226533
+ * @summary Ensure that -XX:+BootstrapJVMCI does not trigger an assertion
+ * @requires vm.jvmci
```

Comment:
@jerboaa Hello, does this `@requires` make sense? Should I put something like `vm.graal.enabled` in there too?

Reply:
Yes, `vm.graal.enabled` would make sense for this test.

Reply:
@jerboaa 
With this, the test runs:
```
 * @requires vm.jvmci & !vm.graal.enabled
```
It is exlcuded with:
```
 * @requires vm.jvmci & vm.graal.enabled
```
So....not sure what to make of `vm.graal.enabled` then.

My build:
```
 ./configure --with-zlib=bundled --with-libjpeg=bundled --with-libpng=bundled \
  --with-lcms=bundled --with-giflib=bundled --with-stdc++lib=static \
  --disable-warnings-as-errors --with-boot-jdk=/home/karm/X/JDKs/openjdk-11 \
  --enable-unlimited-crypto --with-jvm-features=zgc,shenandoahgc 
```

Reply:
Sorry this property checks whether or not the Graal JIT is enabled at runtime i.e. via a command line flag when the test run boots up. Since we don't need the graal JIT to run the test (the forked JVM enables the graal JIT), `@requires vm.jvmci` should be sufficient. Sorry again, I confused this with the graal JVM feature being compiled in.

Reply:
Ack.